woocommerce/google-listings-and-ads
# Internalize campaign creation and editing to `AdsCampaign` to eliminate the associated *FormContent components

### Changes proposed in this Pull Request:

It's related to https://github.com/woocommerce/google-listings-and-ads/pull/1799#discussion_r1038754145

- Internalize campaign creation and editing to `AdsCampaign`, so the associated **FormContent* components could be removed
   - `CreateCampaignFormContent`
   - `EditPaidAdsCampaignFormContent`
- Remove the almost duplicate component `setup-ads/ads-stepper/create-campaign/index.js` and replace with `AdsCampaign`
- Merge `setup-ads/setup-ads-form.js` and `setup-ads/setup-ads-form-content.js` because the latter only combined two components and was not reused.

This way, it could reduce some naming ambiguities and duplicate code for campaign-related components.


### Detailed test instructions:

1. Go to the related pages
   - Step 4 of onboarding - `/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fsetup-mc`
   - Ads setup - `/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fsetup-ads`
   - Campaign creation - `/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fdashboard&subpath=%2Fcampaigns%2Fcreate`
   - Campaign editing - `/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fdashboard&subpath=%2Fcampaigns%2Fedit&programId={CAMPAIGN_ID}`
2. Check if the UI and functionality are the same
   - 💡 The `margin-right` of the currency label is different because it was missed to apply the consistent style while developing the Streamlined FL+Paid Ads project.
   ![2022-12-05 下午2 08 57](https://user-images.githubusercontent.com/17420811/205561715-f50dede9-24e7-4def-a618-9495242cd0ed.png)
   - 💡 The `CampaignPreviewCard` is expected to show on all campaign creation and editing pages except for the onboarding.
   ![image](https://user-images.githubusercontent.com/17420811/205604915-774927f3-e7bf-4a4f-be06-3069fb7c6fe2.png)

### Changelog entry
